client/src/components/ReviewList.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+
+class ReviewList extends React.Component {
+  constructor(props) {
+    super(props);
+    // this.state = {
+    //   review: '',
+    // };
+    // this.pickProperty = this.pickProperty.bind(this);
+  }
+
+  componentDidMount() {
+    // this.setState({review: this.props.reviews});
+    console.log(this.props);
+  }
+
+  render() {
+    if(!this.props.reviews) {
+      return (
+        <div>
+          <div>
+            <span>Avatar</span>
+            <span>username</span>
+            <span>Day Posted</span>
+          </div>
+          <div>Review...</div>
+        </div>
+      );
```

Comment:
what's the point of an empty review as opposed to no review? or am I misunderstanding what this code means.

Reply:
It was just a place holder to see if certain parts of my review were updating properly -> though for the most part I didn't end up changing it because of a lack of time